HISTORY.rst

```diff
@@ -3,6 +3,140 @@
 Release History
 ---------------
 
+1.0.0 (2022-06-30) Quartzsite
++++++++++++++++++++++++++++++++++++++++++++
+
+**Improvements**
+
+* feat: Implement auto-conversion of function to TextElement #1346 
+* Readme: Add Matrix badge and description #1164
+* examples: Convert nodes to list when drawing random sample#1330
+* examples: Use nicer color for bank_reserves #1324 
+* examples: Use nicer color for chart #1323 
+* model: Implement initialize_data_collector #1287
+* Default batch_run to 1 CPU #1300 
```

Comment:
This is currently under dispute on whether it is a breaking change or not. @Corvince said it is not. I said it is.

Reply:
What about "Potentially breaking change"?

Reply:
Sounds good to me.